storage/client_event_test.go

```diff
@@ -33,12 +35,14 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+const numberOfProtoMethods = 30
+
 type storeEventReader struct {
 	// If true, Update events will be recorded in full detail. If false, only a
 	// count of update events will be recorded.
 	recordUpdateDetail  bool
-	perStoreFeeds       map[proto.StoreID][]string
-	perStoreUpdateCount map[proto.StoreID]map[proto.Method]int
+	perStoreFeeds       [][]string
+	perStoreUpdateCount [][numberOfProtoMethods]int
```

Comment:
I feel like sorting the keys is seriously a lot of shenanigans, but i'll try it tomorrow